tests/draft7/optional/content.json

```diff
@@ -0,0 +1,62 @@
+[
+    {
+        "description": "validation of string-encoded content based on media type",
+        "schema": {
+            "contentMediaType": "application/json"
```

Comment:
Hmmmm I probably have to read the spec now I assume, but wow, this seems rather large -- are we gonna ask implementors to be able to validate that something is a valid `video/mp4`?

Reply:
@Julian As with `format`, doing validation for `contentMediaType` and `contentEncoding` is entirely options.  I suspect many systems will use them as annotations (instructions to the application using the schema and instance) rather than assertions (affecting a boolean outcome, in this case validation).  The assertion vs annotation thing is formalized in the new core and validation specs.

Unlike `format`, which has a set of values that, if you implement validation for the keyword at all, you SHOULD implement that set of values, with `content*` there is no requirement for any particular set of encodings or media types to be validated.  I assume that many validators will provide a generic hook and allow extensions to register handlers for particular media types an/or encodings, as is often done with custom formats already.

At least, that was the intention.

Reply:
Got it -- makes sense, and I didn't notice that this was in optional/ so thanks for pointing that out.